includes/wikia/models/tests/DesignSystemSharedLinksTest.php

```diff
@@ -9,7 +9,9 @@ class DesignSystemSharedLinksTest extends WikiaBaseTest {
 	 * @param $hrefs hrefs definition in different languages
 	 * @param $expectedResult
 	 */
-	public function testGetHref( $lang, $hrefs, $expectedResult ) {
+	public function testGetHref( $environment, $lang, $hrefs, $expectedResult ) {
+		$this->mockEnvironment( $environment );
+
 		DesignSystemSharedLinks::getInstance()->setHrefs( $hrefs );
 
 		$result = DesignSystemSharedLinks::getInstance()->getHref( 'create-new-wiki', $lang );
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "create-new-wiki" 20 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)